.gitattributes

```diff
@@ -0,0 +1,42 @@
+*.pbxproj binary merge=union
+
+# Consistent line endings for text files
+*.c text
+*.cpp text
+*.h text
+*.py text eol=lf
```

Comment:
Either remove it in favour of the suggestion I posted below or add the `diff=python` here and remove the line for .py from my suggestions.